frontend/src/models/api/Property.ts

```diff
@@ -0,0 +1,42 @@
+import { Api_Address } from './Address';
+import { Api_ConcurrentVersion } from './ConcurrentVersion';
+import Api_TypeCode from './TypeCode';
+
+export interface Api_Property extends Api_ConcurrentVersion {
+  id: number;
```

Comment:
Similar to other values below. Since we don't promise non-nullability on the backend API, this should not expect it.

Reply:
For properties we currently do not support CREATE, only READ and UPDATE as they are brought into the system via ETL. So that's why I left the id as non-null as it is guaranteed (for now). 

I will turn it into nullable for consistency but note that we don't plan on creating properties for now so it will always come back set to a value